vmcnet/train/runners.py

```diff
@@ -582,6 +582,18 @@ def run_molecule() -> None:
     """Run VMC on a molecule."""
     reload_config, config = train.parse_config_flags.parse_flags(FLAGS)
 
+    reload = (
+        reload_config.logdir != train.default_config.NO_RELOAD_LOG_DIR
+        and reload_config.use_checkpoint_file
+    )
+    reload_run_from_checkpoint = reload and not reload_config.model_only
+    reload_model_from_checkpoint = reload and reload_config.model_only
+
+    if reload_model_from_checkpoint:
+        config.notes = config.notes + " (reloaded model from {}/{})".format(
```

Comment:
This is nice; can you also add handling for the reload_run_from_checkpoint case?